components/notificacoes.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Componentes de Notificação</title>
+    <link rel="stylesheet" href="css/components/notificacoes.css"/>
```

Comment:
O arquivo html `notificacoes.html` está na pasta `components`. O arquivo css `notificacoes.css` está na pasta css, por esse motivo o caminho correto para o arquivo CSS seria:
```suggestion
    <link rel="stylesheet" href="../css/components/notificacoes.css"/>
```